src/MediaUpload.js

```diff
@@ -38,6 +38,30 @@ const styles = theme => ({
 
 });
 
+const dbxImageOptions = {
+    success: function(files) {
+        alert("Here's the file link: " + files[0].link)
+    },
+    cancel: function() { },
+    linkType: "preview", // or "direct"
+    multiselect: false,
+    extensions: ['images'],
+    folderselect: false,
+    sizeLimit: 1024 * 1024 * 1024, // in bytes
+};
+
+const dbxAudioOptions = {
+    success: function(files) {
+        alert("Here's the file link: " + files[0].link)
+    },
+    cancel: function() { },
```

Comment:
Is this property required or can we just omit it if it's an empty function? Can you use a lambda here for consistency?